doc/release-notes/9356-rate-limiting.md

```diff
@@ -0,0 +1,12 @@
+Rate limiting has been added to prevent users from over taxing the system either deliberately or by runaway automated processes. Rate limiting can be configured on a tier level with tier 0 being reserved for guest users and tiers 1-any for authenticated users. Administrator/Superuser accounts are exempt from rate limiting.
```

Comment:
```suggestion
Rate limiting has been added to prevent users from over taxing the system either deliberately or by runaway automated processes. Rate limiting can be configured on a tier level with tier 0 being reserved for guest users and tiers 1-any for authenticated users. Superuser accounts are exempt from rate limiting.
```

It's more clear to only say superuser.